### ISSUE TITLE
Error installing v1 pre-release on WSL2 Ubuntu 20.04.5 LTS

### ISSUE THREAD

**Issue by geealbers** [thegetty/quire#627]
### Discussed in https://github.com/thegetty/quire/discussions/622

<div type='discussions-op-text'>

<sup>Originally posted by **chrisdaaz** December  7, 2022</sup>
I'm seeing the following error after running `quire --version` from [the v1 getting started (macOS) documentation](https://quire.getty.edu/docs-v1/install-uninstall/#uninstall-quire):

```sh
/usr/bin/env: ‘node --no-warnings’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

I can't tell if this is a problem with my versions of `node` / `npm` or my system (`Ubuntu 20.04.5 LTS` on Windows Subsystem for Linux). Maybe there's something I can change in the CLI package?

Here's the full output of my terminal:

```sh
chris @ ~ $ node --version
v18.12.1
chris @ ~ $ npm --version
8.19.2
chris @ ~ $ npm install --global @thegetty/quire-cli
npm WARN deprecated xmldom@0.1.31: Deprecated due to CVE-2021-21366 resolved in 0.5.0
npm WARN deprecated @babel/polyfill@7.12.1: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
npm WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.

added 393 packages, and audited 830 packages in 16s

101 packages are looking for funding
  run `npm fund` for details

13 vulnerabilities (2 low, 7 moderate, 3 high, 1 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
chris @ ~ $ quire --version
/usr/bin/env: ‘node --no-warnings’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
chris @ ~ $ npm list --global
/usr/lib
├── @thegetty/quire-cli@1.0.0-rc.0
├── corepack@0.14.2
└── npm@8.19.2
```

Any suggestions? Happy use an older version, but eager to try v1!</div>

**Comment by geealbers** [comment]
In progress for v1.0.0-rc.1: https://jira.getty.edu/browse/DEV-13838

**Comment by Erin-Cecele** [comment]
Closed with PR #629
